### PR TITLE
Reduce GOGC while fuzzing

### PR DESCRIPTION
**Pull request by Kubuxu**
I've run out of memory when running 60s fuzzing

### CODE REVIEW

**Comment by Kubuxu** [comment]
> Fuzz failure on CI is interesting. Can you update your branch and push just to check for false negatives please? things might have been fixed already on main.

I think we are right on the edge of performance with these fuzzy tests which might cause the failure.

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,7 @@ test/cover: GOTEST_ARGS=-coverprofile=coverage.txt -covermode=atomic -coverpkg=.
 .PHONY: test/cover
 
 fuzz: FUZZTIME ?= 10s # The duration to run fuzz testing, default to 10s if unset.
+fuzz: GOGC ?= 400 # Reduce GC frequency during testing, default to 200 if unset.
```
Comment (masih) [inline Makefile:16]:
```suggestion
fuzz: GOGC ?= 400 # Reduce GC frequency during testing, default to 400 if unset.
```

**Comment by Kubuxu** [comment]
I tested with GOGC=100 and fuzzing still fails.

**Review by masih — APPROVED** [review]

**Comment by Kubuxu** [comment]
```
--- FAIL: FuzzHonestMultiInstance_SyncAgreement (15.29s)
    fuzzing process hung or terminated unexpectedly: exit status 2
    Failing input written to testdata/fuzz/FuzzHonestMultiInstance_SyncAgreement/1c9eda1b22bf9e69
    To re-run:
    go test -run=FuzzHonestMultiInstance_SyncAgreement/1c9eda1b22bf9e69
```

**Pull request merged**